go.mod

```diff
@@ -15,5 +15,6 @@ require (
 	k8s.io/api v0.0.0-20190409021203-6e4e0e4f393b
 	k8s.io/apimachinery v0.0.0-20190404173353-6a84e37a896d
 	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
-	k8s.io/klog v0.3.0
+	k8s.io/code-generator v0.15.8
```

Comment:
`client-go`, `api`, and `apimachinery` also need to be bumped to 1.15-compatible versions. Then you have to rerun `./generate-client.sh` and commit those changes.

Reply:
sure lets

Reply:
so i did that, and no new changes. still checking in a generate commit with just the chmod change to make it work

Reply:
Don't forget to run `go mod vendor`, otherwise it'll still be executing `code-generator` 1.14.

I don't like that `vendor` directory -- too messy!